fre/yamltools/data_table/combine_data_table_yamls.py

```diff
@@ -118,4 +118,4 @@ def combine_data_table_yaml(in_files, out_file, force):
 
 
 if __name__ == "__main__":
```

Comment:
Just FYI: I don't think the data_table and diag_table yamltools are really integrated in the fre-cli yet. 

They live here: https://github.com/NOAA-GFDL/fms_yaml_tools/tree/main/fms_yaml_tools

We added them in the fre-cli with the intent to add them in but I don't think anything else was done with them yet.

Reply:
fair enough! we need to make `fms_yaml_tools` a proper dependency and call ala `catalogbuilder`